castle/cms/profiles/default/registry/resources.xml

```diff
@@ -223,4 +223,9 @@
     </value>
   </record>
 
+  <records prefix="plone.resources/slideshow-js"
```

Comment:
We'll need to add this to a new upgrade step too so we don't have to re-run default profile on existing sites

Reply:
k, i'll do that now